das/redundant_storage_service.go

```diff
@@ -119,25 +121,43 @@ func (r *RedundantStorageService) Close(ctx context.Context) error {
 	return anyError
 }
 
-func (r *RedundantStorageService) ExpirationPolicy(ctx context.Context) ExpirationPolicy {
+func (r *RedundantStorageService) ExpirationPolicy(ctx context.Context) (arbstate.ExpirationPolicy, error) {
 	// If at least one inner service has KeepForever,
 	// then whole redundant service can serve after timeout.
 	for _, serv := range r.innerServices {
```

Comment:
Similarly, can the three loops here be combined?

Reply:
Done.